lib/cook_book.rb

```diff
@@ -0,0 +1,49 @@
+require 'date'
+
+class CookBook
+  attr_reader :recipes,
+              :date
+
+  def initialize
+   @recipes = []
+   @date = Date.today
```

Comment:
This doesn't return the date in the format asked. Your test didn't catch it because you stubbed out the cookbook#date method instead of the Date.today. Close though!